### DISCUSSION TITLE
Catenary Elements

### DISCUSSION THREAD

**Discussion by sonideft** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#283] [Ideas]
Just wondering if there is a current way to define/use Catenary Cable elements, or if there was interest in creating such a thing?

**Comment by sonkaste17** [comment]
Hello,
there is no way how to write down cables in SAF. Cables, in general, are objects used in structural analysis and therefore cables are candidates for SAF extension for sure.

Are you interested in having cables in SAF?

**Reply by sonideft** [reply]
I am very much interested in having Cables in SAF. I deal with Cable Elements strung between structures and structure elements all the time and would like the option of transferring models to other tools for verification and sharing purposes. Hopefully there are others interested, I could certainly contribute.

**Reply by sonkaste17** [reply]
That is great! We are very happy for help with the design of new SAF objects. The first thing to start with is defining key properties that we should cover in SAF.

**Reply by sonideft** [reply]
I will create a proposal then in the next few weeks.

**Comment by sonideft** [comment]
I have an question first on the best approach to get as many other users of SAF adopting the proposed Cable Element. I could propose a simple, one metal/material model assuming linear elastic behavior. This alone could address a great number of common scenarios. If the model includes fields for unstretched length and %permanent strain/stretch (plastic behavior), the same element could also be used in cases where permanent stretch can be estimated by other means.

A more advanced cable element could be proposed that could include nonlinear elastic behavior of the one metal/material with the stress/strain curve being defined with polynomial coefficients. In this case the user can estimate the permanent stretch that could result from certain load cases, while also considering linear elastic behavior if loads are less than the one that determined the permanent stretch.

A third Cable Element could be proposed that includes two metals/materials with nonlinear elastic behavior. This is actually quite common in utility pole structure scenarios. I'm not sure if it has other use cases.

My question then would be if there should be more than one cable element proposed? All could be achieved with one element with certain fields set at 0.0. However, it would be less obvious at first glance to what extent a particular software fully supports each layer of complexity. Again, I'm hoping to create a Cable Element that others can build support for.

Any thoughts?

**Comment by sonideft** [comment]
Perhaps I should propose something basic first.
